examples/rename_brainvision_files.py

```diff
@@ -1,44 +1,60 @@
 """
-=====================================
-06. Rename BrainVision EEG data files
-=====================================
+======================================================
+06. Rename and/or anonymize BrainVision EEG data files
+======================================================
 
-The BrainVision file format is one of the recommended formats to store EEG data
-within a BIDS directory. To organize EEG data in BIDS format, it is often
-necessary to rename the files. In the case of BrainVision files, we would have
-to rename multiple files for each dataset instance (i.e., once per recording):
+According to the EEG extension to BIDS [1]_, the `BrainVision data format`_ is
+one of the recommended formats to store EEG data within a BIDS directory.
+
+To organize EEG data in BIDS format, it is often necessary to rename the files.
+In the case of BrainVision files, we would have to rename multiple files for
+each dataset instance (i.e., once per recording):
```

Comment:
```suggestion
each recording:
```
I think it's easier to understand that way?